Realm/RLMUsernamePasswordProviderClient.h

```diff
@@ -96,12 +98,12 @@ NS_ASSUME_NONNULL_BEGIN
  
  @param email  The email address of the user.
  @param password The desired new password.
- @param args A pre-serialized list of arguments. Must be a JSON array.
+ @param args A pre-serialized list of arguments passed in as a BSON array.
  @param completionHandler A callback to be invoked once the call is complete.
 */
 - (void)callResetPasswordFunction:(NSString *)email
                          password:(NSString *)password
-                             args:(NSString *)args
+                             args:(NSArray<NSDictionary<NSString *, id<RLMBSON>> *> *)args
```

Comment:
This should not be an `NSArray` of documents. In this case, it should match callFunction: `NSArray<id<RLMBSON>> *`